indico/modules/events/requests/templates/event_request_details.html

```diff
@@ -5,10 +5,17 @@
 
 {% set has_req = req and not req.state.name == 'withdrawn' %}
 {% set show_management = is_manager and has_req %}
+{%- if not req -%}
```

Comment:
missing i18n on the sets. you could either use `_()` in there or use a single set block:

```
{% set button_title -%}
    {%- if not req -%}
        {%- trans %}Send request{% endtrans -%}
    {%- elif ... -%}
        ...
    {%- endif -%}
{%- endset %}
```